egasub/submission/init.py

```diff
@@ -3,15 +3,14 @@
 import yaml
 import json
 from ..ega.entities import Dac, Policy, Contact, Study, Submission, SubmissionSubsetData
-from ..ega.services import login, logout, object_submission, query_by_type, prepare_submission
```

Comment:
We should do a 'logout' here after `initialize_dac_policy_study` is done it's work. So the solution is to keep 'logout' import, and call 'logout' at the end of `initialize_dac_policy_study` function.